src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -507,8 +520,9 @@ def connection_create_func(cmd, client,  # pylint: disable=too-many-locals,too-m
 
     validate_service_state(parameters)
     if enable_mi_for_db_linker and auth_action != 'optOutAllAuth':
+        logger.warning('Start creating mi for db linker')
```

Comment:
if the message is intended to show to user, pls make it official and professional,  Start creating managed identity for fabric sql .blahblah...

Reply:
removed as this is not necessary information